src/System.Runtime.Numerics/tests/BigInteger/PerformanceTests.cs

```diff
@@ -21,7 +21,7 @@ public PerformanceTests(ITestOutputHelper output)
         }
 
         [Theory]
-        [ActiveIssue("PerformanceTest")]
```

Comment:
I misspoke about these performance test ones -- they don't need an issue. please change to
```
[Theory(Skip = "Performance test")]
```

They don't need ActiveIssue.